src/algorithms/cochran.ts

```diff
@@ -0,0 +1,43 @@
+import { CochranResult } from '../types';
+import { standardDeviation } from 'simple-statistics';
+import criticalValueTable from './../grubbs/criticalValueTable';
+
+export function Cochran(values: Array<number[]>): CochranResult {
+  /* p -> numune sayısı  */
+  const pValue = values.length;
+
+  /* n -> yapılan tekrar,test sayısı */
+  const nValue = values[0].length;
```

Comment:
We need to use this value to find the corresponding critical value from this table:

<img width="607" alt="Screenshot 2022-06-22 at 11 48 53" src="https://user-images.githubusercontent.com/735011/174986332-f0b097e8-156e-4111-90d0-631d9f84c5c0.png">

Reply:
- [ ] After creating the new critical table, I will change the codes here.